src/styles/blocks/card.scss

```diff
@@ -0,0 +1,116 @@
+@import "../../layout_stars/src/style.css";

```

Comment:
import every import to the main.scss file
Use it in all cases
```suggestion
```

Reply:
Same as that one